### DISCUSSION TITLE
[Domoticz] Backend parser when retrieving data from Domoticz

### DISCUSSION THREAD

**Discussion by marclemmens** [grafana/grafana-infinity-datasource#502] [Show and tell]
Hello there,

I like your datasource A LOT. It works as expected when I use the default parser but when I try to use Alerts I have to switch to the backend parser and I don't see any data. 
Can you point me in the right direction what I could test?
The data looks like:
0
2023-01-01 09:00:00
0
2023-01-01 09:05:00
0
2023-01-01 09:10:00
0
2023-01-01 09:15:00
0
2023-01-01 09:20:00
0
2023-01-01 09:25:00
83
2023-01-01 09:30:00
65
2023-01-01 09:35:00

Many thanks in advance.

Kind regards,

Marc Lemmens
Belgium

**Comment by yesoreyeram** [comment]
Can you provide more details. What is original data looks like ? What you are trying to achieve etc? Json/ CSV/XML? Actual query json and output of curl is recommended
Probably root selector might need fix or field types needs fix.

**Comment by marclemmens** [comment]
hello yesoreyeram,

meanwhile I solved it by entering the root of the data (result).
Thanks for investigating and answering so quickly.

Kind regards,

Marc Lemmens

<img width="2240" alt="Scherm­afbeelding 2023-01-08 om 09 11 56" src="https://user-images.githubusercontent.com/45148562/211186647-f650e0ef-7453-4af2-bd9a-73a946759c71.png">

<img width="2240" alt="Scherm­afbeelding 2023-01-08 om 09 14 09" src="https://user-images.githubusercontent.com/45148562/211186684-c2273f84-69e0-47fe-aa88-b49f34b89da5.png">
